sdk/containerregistry/azure-containerregistry/tests/testcase.py

```diff
@@ -121,36 +108,47 @@ def get_audience(authority: str) -> str:
         logger.warning("US Gov cloud auth audience")
         return "https://management.usgovcloudapi.net"
 
-def import_image(authority, repository, tags, is_anonymous=False):
-    logger.warning(f"Import image authority: {authority}")
-    if is_anonymous:
-        registry_name = os.environ.get("CONTAINERREGISTRY_ANONREGISTRY_NAME")
-    else:
-        registry_name = os.environ.get("CONTAINERREGISTRY_REGISTRY_NAME")
-    sub_id = os.environ.get("CONTAINERREGISTRY_SUBSCRIPTION_ID")
-    tenant_id=os.environ.get("CONTAINERREGISTRY_TENANT_ID")
-    client_id=os.environ.get("CONTAINERREGISTRY_CLIENT_ID")
-    client_secret=os.environ.get("CONTAINERREGISTRY_CLIENT_SECRET")
-    credential = ClientSecretCredential(
-        tenant_id=tenant_id, client_id=client_id, client_secret=client_secret, authority=authority
-    )
-    audience = get_audience(authority)
-    scope = [audience + "/.default"]
-    mgmt_client = ContainerRegistryManagementClient(
-        credential, sub_id, api_version="2019-05-01", base_url=audience, credential_scopes=scope
-    )
-    logger.warning(f"LOGGING: {sub_id}{tenant_id}")
-    registry_uri = "registry.hub.docker.com"
-    rg_name = os.environ.get("CONTAINERREGISTRY_RESOURCE_GROUP")
-
-    import_source = ImportSource(source_image=repository, registry_uri=registry_uri)
-
-    import_params = ImportImageParameters(mode=ImportMode.Force, source=import_source, target_tags=tags)
-
-    result = mgmt_client.registries.begin_import_image(
-        rg_name,
-        registry_name,
-        parameters=import_params,
+def get_credential(authority: str, **kwargs):
```

Comment:
Nit: I would consider changing this method name to differentiate it from the test class instance method; not a big deal though

Reply:
I think we can use the same name because we are not using the method in parent class in elsewhere in this package. This one is actually an `override`.